get_value_path.c

```diff
@@ -0,0 +1,28 @@
+#include "shell.h"
+/**
+ * path_value - function that find the value of the path
+ * Return: pointer to the value of path or NULL if unsuccesfull
+ */
+char *path_value(void)
```

Comment:
its better the function name looks similar to getenv somethng like _getenv so we dont get confused to what it does or have a hard time looking for it when we want to call it somewhere else.. and its best the function takes a parameter instead of void, so you can call it with _getenv(PATH) rather than _getenv()